packages/ui/src/Button/index.tsx

```diff
@@ -8,10 +8,13 @@ import { LucideIcon } from 'lucide-react';
 import { Density } from '../types/Density';
 import { useDensity } from '../hooks/useDensity';
 import { ActionType } from '../utils/ActionType';
+import { MotionProp } from '../utils/MotionProp';
+import { motion } from 'framer-motion';
 
 const iconOnlyAdornment = css<StyledButtonProps>`
   border-radius: ${props => props.theme.borderRadius.full};
   padding: ${props => props.theme.spacing(1)};
+  width: max-content;
```

Comment:
Fixes a weird issue where the button was sometimes full-width when it shouldn't have been